docs/change-log/index.md

```diff
@@ -169,7 +342,7 @@ API-client connections. [View in Git](https://github.com/mlrun/mlrun/pull/2613).
 improve handling of high workloads against the MLRun DB. You can configure the number of workers for an MLRun 
 service, which is applied to the service's user-created pods. The default is 2. 
    - v1.1.0 cannot run on top of 3.0.x.
-   - For Iguazio v <3.5.0 number of workers set to 1 by default. To change this number, contact support (helm-chart change required).
+   - For Iguazio <v3.5.0 number of workers set to 1 by default. To change this number, contact support (helm-chart change required).
```

Comment:
```suggestion
   - For Iguazio versions older than 3.5.0, the number of workers is set to 1 by default. To change this number, contact support (helm-chart change required).
```

Reply:
changed to:
 For Iguazio versions prior to v3.5.0, the number of workers is set to 1 by default.